src/App.css

```diff
@@ -1,6 +1,149 @@
+:root {
+  --primary-color: #a33846;
+  --secondary-color: #b61e1e;
+  --accent-color: #90313e;
+  --background-color: #d6848c;
+  --text-color: #040102;
+}
+
+.wrapper {
+  max-width: 1000px;
+  display: flex;
+  flex-direction: column;
+}
+
+.wrappper {
+  max-width: 1000px;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  padding: 10px 10px;
+}
+
+.navbar-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 7.5vh;
+  min-height: 70px;
+  background-color: var(--background-color);
+  box-shadow: 0 5px 30px #bebebe;
+}
+
+.navbar-wrapper {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.link-wrapper {
+  display: flex;
+  justify-content: end;
+  gap: 30px;
+  align-items: center;
+  width: 40%;
+}
+
+.link-wrapper a {
+  text-decoration: none;
+  color: var(--text-color);
+  padding: 11px 18px;
+  border-radius: 12px;
+  position: relative;
+  z-index: 30;
+}
+
+.link-wrapper a::before {
+  content: '';
+  position: absolute;
+  z-index: -1;
+  bottom: 10%;
+  left: 0;
+  width: 100%;
+  height: 10%;
+  background-color: var(--secondary-color);
+  transition: all 0.3s ease;
+}
+
+.link-wrapper a:hover::before,
+.link-wrapper a.active::before {
+  height: 10%;
+  content: '';
+  position: absolute;
+  z-index: -1;
+  bottom: 10%;
+  left: 0;
+  width: 100%;
+  background-color: blue;
+  transition: all 0.4s ease;
+}
+
 .App {
   text-align: center;
   height: 100vh;
   display: grid;
   place-items: center;
 }
+
+.header {
+  display: flex;
+}
+
+.calcu-container {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  height: 100vh;
+  padding: 20px;
+}
+
+.calc-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  min-width: 300px;
+  text-align: center;
+}
```

Comment:
- [ ] The changes you made have affected the layout of the calculator page design. Kindly make sure to align the contents of the `Calculator` page and make sure to remove the horizontal scroll on this page in order to match the provided wireframe.

   - ### YOUR DESIGN
   ![chrome-capture-2023-6-11 (1)](https://github.com/abdozayan12/Math-Magicians/assets/87197412/67e8dc5a-6924-4f4c-bd64-72bfd86645cc)

   - ### PROVIDED WIREFRAME
   ![image](https://github.com/abdozayan12/Math-Magicians/assets/87197412/a345e79c-6f14-4400-b2c8-7efb09b11a36)